README.md

```diff
@@ -54,198 +69,108 @@ This CLI tool gathers information from your system and generates a cloud-config
+ If interactive mode is enabled, the script will prompt for the necessary information to     
+ generate the cloud-init config file and step through each configuration module and show the 
+ cloud-config portion generated by that module.                                              
+ Only -f can be used with -i/--interactive                                                   
+                                                                                             
+╭─ Options ─────────────────────────────────────────────────────────────────────────────────╮
+│ --interactive            -i        Enable interactive mode.                               │
+│ --quiet                  -q        Enable quiet output. Only critical errors and          │
+│                                    essential information will be displayed.               │
+│ --output-path            -o  TEXT  Path to output file. [default: cloud-config.yaml]      │
+│ --force                  -f        Write over output file if it already exists.           │
+│ --enable-hostname                  Enable gathering the hostname of the machine. This     │
+│                                    will cause issues unless this exact machine is being   │
+│                                    redeployed using the generated cloud-init config.      │
+│ --gather-public-keys               Enable gathering of all public key files in the ~/.ssh │
+│                                    directory. This will allow you to use the same public  │
+│                                    keys on the new machine as the current machine.        │
+│ --password                   TEXT  Set the password for the user. WARNING: This is        │
+│                                    incredibly insecure and is stored in plaintext in the  │
+│                                    cloud-init config.                                     │
+│ --disable-apt                      Disable the gathering and generation of apt config.    │
+│ --disable-snap                     Disable the gathering and generation of snap config.   │
+│ --disable-ssh                      Disable the gathering and generation of ssh config.    │
+│ --disable-user                     Disable the gathering and generation of user config.   │
+│ --rename-to-ubuntu-user            Keep the current user config but rename it to the      │
+│                                    default 'ubuntu' user.                                 │
+│ --help                   -h        Show this message and exit.                            │
+│ --version                          Show the version and exit.                             │
+╰───────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
 ### Example Calls:
 
 #### Minimum Call
 ```bash
-python -m not_cloud_init.cli generate
+cc-builder
 ```
 This will output a cloud-config file named cloud-config.yaml with the default settings.
```

Comment:
super nit: Move the paragraph about the command, above the command instead of below. Something like:

```
### heading
paragraph
<command>
```

Reply:
agreed! thanks king